### PR TITLE
Rnafusion 2.0.0!

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

As it is a major refacturing, check the CHANGELOG for details. Usage is also updated!

- [x] This comment contains a description of changes (with reason) 
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If necessary, also make a PR on the [nf-core/rnafusion branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/rnafusion)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [x] `README.md` is updated

### CODE REVIEW

**Review by matq007 — APPROVED** [review]
LGTM 👍🏻

File: README.md [hunk 1]
```diff
@@ -1,130 +1,146 @@
...
+   - [FusionCatcher](https://github.com/ndaniel/fusioncatcher) fusion detection
+9. Fusion-report subworkflow
+   - Merge all fusions detected by the different tools
+   - [Fusion-report](https://github.com/matq007/fusion-report)
+10. FusionInspector subworkflow
+    - [FusionInspector](https://github.com/FusionInspector/FusionInspector)
+11. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+12. QC for mapped reads ([`QualiMap: BAM QC`](https://kokonech.github.io/qualimap/HG00096.chr20_bamqc/qualimapReport.html))
+13. Index mapped reads ([samtools index`](http://www.htslib.org/))
+14. Collect metrics ([picard CollectRnaSeqMetrics`](https://gatk.broadinstitute.org/hc/en-us/articles/360037057492-CollectRnaSeqMetrics-Picard-)) and ([picard MarkDuplicates`](https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-))
```
Comment (matq007) [inline README.md:87]:
```suggestion
14. Collect metrics ([`picard CollectRnaSeqMetrics`](https://gatk.broadinstitute.org/hc/en-us/articles/360037057492-CollectRnaSeqMetrics-Picard-) and ([`picard MarkDuplicates`](https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-))
```

File: README.md [hunk 2]
```diff
@@ -1,130 +1,146 @@
-# ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo.png)
+# ![nf-core/rnafusion](docs/images/nf-core/rnafusion_logo_light.png#gh-light-mode-only) ![nf-core/rnafusion](docs/images/nf-core/rnafusion_logo_dark.png#gh-dark-mode-only)
```
Comment (matq007) [inline README.md:1]:
```suggestion
# ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo_light.png#gh-light-mode-only) ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo_dark.png#gh-dark-mode-only)
```

Comment (maxulysse) [reply README.md:1]: That's fixed in nf-core tools dev, good catch, I always forgot to check this one out

File: README.md [hunk 3]
```diff
@@ -1,130 +1,146 @@
...
+
+#### Main workflow
+
+1. Input samplesheet check
+2. Concatenate fastq files per sample
+3. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+4. Arriba subworkflow
+   - STAR alignment
+   - Samtool sort
+   - Samtool index
```
Comment (matq007) [inline README.md:62]:
```suggestion
    - [STAR](https://github.com/alexdobin/STAR) alignment
    - [Samtool](https://github.com/samtools/samtools) sort
    - [Samtool](https://github.com/samtools/samtools) index
```

File: README.md [hunk 4]
```diff
@@ -1,130 +1,146 @@
...
+8. Fusioncatcher subworkflow
+   - [FusionCatcher](https://github.com/ndaniel/fusioncatcher) fusion detection
+9. Fusion-report subworkflow
+   - Merge all fusions detected by the different tools
+   - [Fusion-report](https://github.com/matq007/fusion-report)
+10. FusionInspector subworkflow
+    - [FusionInspector](https://github.com/FusionInspector/FusionInspector)
+11. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+12. QC for mapped reads ([`QualiMap: BAM QC`](https://kokonech.github.io/qualimap/HG00096.chr20_bamqc/qualimapReport.html))
+13. Index mapped reads ([samtools index`](http://www.htslib.org/))
```
Comment (matq007) [inline README.md:86]:
```suggestion
13. Index mapped reads ([samtools index](http://www.htslib.org/))
```

File: README.md [hunk 5]
```diff
@@ -1,130 +1,146 @@
...
-    * [Download references](docs/references.md)
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-3. [Running the pipeline](docs/usage.md)
-4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+## Credits
 
-Use predefined configuration for desired Institution cluster provided at [nfcore/config](https://github.com/nf-core/configs) repository.
+nf-core/rnafusion was originally written by Martin Proks (matq007), Praveen Raj (@praveenraj2018).
```
Comment (matq007) [inline README.md:122]:
```suggestion
nf-core/rnafusion was originally written by Martin Proks ([@matq007](https://github.com/matq007)), Praveen Raj ([@praveenraj2018](https://github.com/praveenraj2018))
```

File: README.md [hunk 6]
```diff
@@ -1,130 +1,146 @@
...
-* **SQUID: transcriptomic structural variation detection from RNA-seq** Cong Ma, Mingfu Shao and Carl Kingsford, Genome Biology, 2018, doi: [https://doi.org/10.1186/s13059-018-1421-5](https://doi.org/10.1186/s13059-018-1421-5)
-* **Fusion-Inspector** download: [https://github.com/FusionInspector](https://github.com/FusionInspector)
-* **fusion-report** download: [https://github.com/matq007/fusion-report](https://github.com/matq007/fusion-report); doi: [https://doi.org/10.5281/zenodo.3520171](https://doi.org/10.5281/zenodo.3520171)
-* **FastQC** download: [https://www.bioinformatics.babraham.ac.uk/projects/fastqc/](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
-* **MultiQC** Ewels, P., Magnusson, M., Lundin, S., & Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics , 32(19), 3047–3048. [https://doi.org/10.1093/bioinformatics/btw354](https://doi.org/10.1093/bioinformatics/btw354) Download: [https://multiqc.info/](https://multiqc.info/)
+- Annick Renevey (@rannick)
```
Comment (matq007) [inline README.md:126]:
```suggestion
We thank the following people for their **extensive assistance** in the development of this pipeline:

- Annick Renevey ([@rannick](https://github.com/rannick))
```

Comment (rannick) [reply README.md]: Adressed in b9c0daf

File: nextflow.config [hunk 7]
```diff
@@ -1,197 +1,253 @@
...
 manifest {
-  name = 'nf-core/rnafusion'
-  author = 'Martin Proks'
-  homePage = 'https://github.com/nf-core/rnafusion'
-  description = 'Nextflow rnafusion analysis pipeline, part of the nf-core community.'
-  mainScript = 'main.nf'
-  nextflowVersion = '>=19.10.0'
-  version = '1.2.0'
+    name            = 'nf-core/rnafusion'
+    author          = 'Martin Proks, @praveenraj2018'
```
Comment (apeltzer) [inline nextflow.config:211]: Looking at who contributed to the pipeline tbh, the handle of Praveen should not be here at all. 

Martin / Maxime / Rickard / Phil and mostly you @rannick shuold be here.

File: README.md [hunk 8]
```diff
@@ -1,130 +1,147 @@
...
-    * [Download references](docs/references.md)
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-3. [Running the pipeline](docs/usage.md)
-4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+## Credits
 
-Use predefined configuration for desired Institution cluster provided at [nfcore/config](https://github.com/nf-core/configs) repository.
+nf-core/rnafusion was originally written by Martin Proks ([@matq007](https://github.com/matq007)), Praveen Raj ([@praveenraj2018](https://github.com/praveenraj2018))
```
Comment (apeltzer) [inline README.md:123]: Was Praveen really doing the effort here? I'm not judging, but the code base tells me very different things.

Comment (apeltzer) [reply README.md:123]: Should have Martin and then Maxime + you Annick here as authors tbh. Others maybe just as a in "acknowledgements" for contributing to testing / running the pipeline / providing feedback (Phil, Me, Rickard, just looking at certain bits / discussion on Slack).

**Review by apeltzer — APPROVED** [review]
Tendency to give this a go, though I have reviewed and tested this now many times before final release now so feel confident we should release now. 

Worst case: Bugfix release 2.0.1, ...

**Review by apeltzer — APPROVED** [review]

**Pull request merged**